docs/config-options/cloud-providers/aws/aws.md

```diff
@@ -144,3 +152,251 @@ If you do not share resources between clusters, you can change the tag to:
 When provisioning a `LoadBalancer` service Kubernetes will attempt to discover the correct subnets, this is also achieved by tags and requires adding additional subnet tags to ensure internet-facing and internal ELBs are created in the correct subnets.
 
 [AWS Documentation: Subnet tagging for load balancers](https://docs.aws.amazon.com/eks/latest/userguide/load-balancing.html#subnet-tagging-for-load-balancers)
+
+### Using the Out-of-Tree AWS Cloud Provider for RKE1 
+
+1. [Node name conventions and other prerequisities ](https://cloud-provider-aws.sigs.k8s.io/prerequisites/) must be followed so that the cloud provider can find the instance. RKE provisioned clusters don't support configuring `providerID`. 
```

Comment:
```suggestion
1. [Node name conventions and other prerequisites ](https://cloud-provider-aws.sigs.k8s.io/prerequisites/) must be followed so that the cloud provider can find the instance. RKE provisioned clusters don't support configuring `providerID`. 
```